yapapi/executor/__init__.py

```diff
@@ -298,18 +299,25 @@ async def start_worker(agreement: rest.market.Agreement) -> None:
             try:
                 act = await activity_api.new_activity(agreement.id)
             except Exception:
-                emit(events.ActivityCreateFailed(agr_id=agreement.id))
+                exc_info = sys.exc_info()
+                assert exc_info[0] is not None and exc_info[1] is not None  # for mypy
```

Comment:
just thinking aloud - wouldn't it be more appropriate to add something like `# type: ignore` rather than assert?

Reply:
You're right, I'm doing it here: https://github.com/golemfactory/yapapi/pull/123/commits/9ea14e4468b521791ad3c49d137d8425fb251fb0 (it's a commit for a separate PR)